site/sidebar.md

```diff
@@ -6,12 +6,59 @@
     - [Binaries](installation/binaries/)
 - [Guides](guides/)
```

Comment:
don't spend too much time on anything related to Guides, given it's being replaced with the book.